button-inc/emissions-elt-demo
# chore: sample insights data

Fixes #14 

## Proposed changes
- Add `json-schema` for Insights/Destination API (completed) response
- Add `json-schema-faker` package for generation of synthetic data
- Add script to generate Destination API response data

### To test
Run the command `yarn run generate:destination-insights` from the `~/app` folder. A `~/app/generated/sample-insights-destination-output.json' file should be created with data matching the `~/jsonschema/destination-api.schema.json` schema format.
